atf_eregs/static/regulations/css/less/module/custom/full-footer.less

```diff
@@ -33,6 +33,14 @@
 		ul li:not(:first-child) {
 			text-align: right;
 		}
+        .footer-small ul li {
+			width: 20%;
+            white-space: nowrap;
+            text-size: 12px;
+        }
+        .footer-small ul li a.external{
+            font-size: 12px;
+        }
```

Comment:
Moved it to the non-size-specific part of the SCSS file, where it does make more sense.